docker/bin/start-worker-reload

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+WATCHED_FILES=$(python -c "print(' '.join([f'/src/{item}' for item in '$DEV_SOURCE_PATH'.split(':')]))")
+exec find $WATCHED_FILES \( -path /src/galaxy_ng/.venv -o -path /src/galaxy_ng/build -o -path /src/galaxy_ng/.eggs \) -prune -o -name '*.py' -o -name '*.env' | entr -n -r pulpcore-worker
```

Comment:
Could this make use of `git ls-files` in lieu of walking the file tree and excluding stuff?

For me at least, I tend to to prefer if tools anything that isn't explicitly in the project. But that is because my source checkouts get littered with all kind of garbage (log files, test-of-something.py, somehelpertool.py, etc)

I know opinion is divided on that expectation though.